istioctl/pkg/multicluster/remote_secret.go

```diff
@@ -68,17 +68,14 @@ func init() {
 }
 
 const (
-	remoteSecretPrefix = "istio-remote-secret-"
-	configSecretName   = "istio-kubeconfig"
-	configSecretKey    = "config"
+	configSecretName = "istio-kubeconfig"
+	configSecretKey  = "config"
 )
 
 func remoteSecretNameFromClusterName(clusterName string) string {
-	return remoteSecretPrefix + clusterName
-}
-
-func clusterNameFromRemoteSecretName(name string) string {
-	return strings.TrimPrefix(name, remoteSecretPrefix)
+	// Replace any "_" with "-" in the cluster name, to conform to k8s resource naming.
```

Comment:
does this break backward compatibility? What is the impact of not renaming? Can you add a release note?